src/lib/evas/common/evas_image_scalecache.c

```diff
@@ -2,12 +2,8 @@
 # include "config.h"
 #endif
 
-#ifdef _WIN32
-# ifndef WIN32_LEAN_AND_MEAN
-#  define WIN32_LEAN_AND_MEAN
-# endif
-# include <windows.h>
-# undef WIN32_LEAN_AND_MEAN
+#ifdef _MSC_VER
```

Comment:
So there is no other thing that `cygwin/msys` were using here?

Reply:
I just checked the blame and prior to `windows.h` it was using `Evil.h`.

Reply:
ohh, so shouldn't this be like?
```
#ifdef _WIN32
 #include <evil_private.h>
Endif
```

Reply:
Yes it should. Thanks